collections/Bodl/MS_Bodl_860.xml

```diff
@@ -62,102 +93,1206 @@
                <msPart xml:id="MS_Bodl_860-part1">
                   <msIdentifier>
                      <altIdentifier type="partial">
-                        <idno type="part">MS. Bodl. 860 – Part 1</idno>
+                        <idno type="part">MS. Bodl. 860, fols. 1–8</idno>
                      </altIdentifier>
                   </msIdentifier>
                   <msContents>
-                     <msItem class="#versus #biblia #commentarii" n="1" xml:id="MS_Bodl_860-part1-item1">
-                        <title key="work_14079" type="desc">Psalm i</title>
-                        <note>(Latin hexameter version, with glosses)</note>
-                        <textLang mainLang="la">Latin</textLang>
+                     <textLang mainLang="la">Latin</textLang>
+                     <msItem class="#versus #biblia #commentarii">
+                        <locus>(fols. 1r-8v)</locus>
+                        <author key="person_61956730">Samuel Presbiter</author>
+                        <title type="desc">Versification of Psalm 1, with marginal glosses</title>
```

Comment:
does this need an authority record?

Reply:
There are no other copies, but I've added the identifiers from Stegmüller and Walther if you think records would be useful.